crates/libs/version/src/lib.rs

```diff
@@ -0,0 +1,132 @@
+#![cfg_attr(not(test), no_std)]
+
+mod bindings;
+use bindings::*;
+
+/// Operating system version information.
+#[derive(Debug, Copy, Clone, PartialEq, Eq, PartialOrd, Ord)]
+pub struct Version {
```

Comment:
I think calling this `OsVersion` might be better? At least, it makes more sense to me when taken out of the context of this crate.

Reply:
Switched to `OsVersion`.